.github/workflows/spark.yaml

```diff
@@ -47,11 +47,17 @@ jobs:
       matrix:
         include:
           - mvn-profile: "datasources-32"
-            spark: "spark-3.2.2"
-            spark-hadoop: "spark-3.2.2-bin-hadoop3.2"
+            spark: "spark-3.2.4"
```

Comment:
Hi, Sem, here I want to confirm that now we support 3.2.x or we just support the specific version 3.2.4? ditto to 3.3.x, 3.4.x and 3.5.x

Reply:
We are supporting 3.2.x, but testing on the latest available (3.2.4)